storage/innobase/fil/fil0fil.cc

```diff
@@ -3425,17 +3313,15 @@ fil_space_t *Fil_shard::space_create(const char *name, space_id_t space_id,
 
   space->is_corrupt = false;
 
-  space->is_space_encrypted = false;
-
-  space->exclude_from_rotation = false;
+  //space->is_space_encrypted = false;
```

Comment:
Sorry, one more question to this file. Can this be removed as well?

Reply:
yes, done.